server/server.js

```diff
@@ -41,19 +41,28 @@ async function handleValidRequest(button, numPresses, batteryLevel) {
     let client = await db.beginTransaction()
 
     let session = await db.getUnrespondedSessionWithButtonId(button.button_id, client)
-        
-    if (session === null) {
-        session = await db.createSession(button.installation_id, button.button_id, button.unit, button.phone_number, numPresses, client)
-    }
-    else {
-        session.incrementButtonPresses(numPresses)
-        await db.saveSession(session, client)
-    }
 
     if ((batteryLevel !== undefined) &&
             (batteryLevel >= 0) &&
             (batteryLevel <= 100)) {
-        await db.saveButtonBatteryLevel(button.button_serial_number, batteryLevel, client)
```

Comment:
You can probably delete `saveButtonBatteryLevel` from `db.js` now.

Reply:
yay red ink red ink red ink red ink *chanting*